sensehat/src/main/java/com/google/android/things/contrib/driver/sensehat/BaroTemp.java

```diff
@@ -0,0 +1,170 @@
+package com.google.android.things.contrib.driver.sensehat;
+
+import com.google.android.things.pio.I2cDevice;
+import com.google.android.things.pio.PeripheralManager;
+
+import java.io.IOException;
+
+/**
+ * This class allows access to the LPS25H on the SenseHat.
```

Comment:
Generally with our sensors we have two classes. One implements the raw protocol, `Lps25h`, and the other plugs into the sensor framework, ie. `Lps25hSensorDriver`.

One example is in our [BMX280](https://github.com/androidthings/contrib-drivers/tree/master/bmx280/src/main/java/com/google/android/things/contrib/driver/bmx280)